implementations/elixir/ockam/ockam_abac/lib/policy_storage/dets.ex

```diff
@@ -0,0 +1,114 @@
+defmodule Ockam.ABAC.PolicyStorage.DETS do
+  @moduledoc """
+  Implementation of Ockam.ABAC.PolicyStorage storing data in DETS table file.
+
+  Runs a gen_server opening the DETS table.
+
+  Options:
+  - filename: atom | string - file name to use for DETS table
+  """
+
+  @behaviour Ockam.ABAC.PolicyStorage
+
+  use GenServer
```

Comment:
All it does here is opens the table, yes. I don't think we need much to add much more.

Reply:
If you have any ideas on better ways to manage ets and dets tables without bringing in something like mnesia - please share.

Reply:
Ah, nevermind, it makes sense to me now. I thought we could simply initialize those tables when we start the application instead of spinning up GenServers, but it would be less configurable and more difficult to manage.